test/e2e/network_test.go

```diff
@@ -812,6 +812,11 @@ func (tc *testContext) createJob(name, image string, command []string, runtimeCl
 			},
 		},
 	}
+	if pullSecret {
+		job.Spec.Template.Spec.ImagePullSecrets = []v1.LocalObjectReference{{
+			Name: "pull-secret",
+		}}
```

Comment:
Was this necessary for the disconnected job to pass? Trying to figure out what the effect of this is

Reply:
yes please see this [test](https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_windows-machine-config-operator/2232/pull-ci-openshift-windows-machine-config-operator-master-vsphere-disconnected-e2e-operator/1805689199450394624) and this [commit](https://github.com/openshift/windows-machine-config-operator/commit/b78917e62a38f3ab0f5ef248ecd9b211886d663e)